corehq/apps/fixtures/models.py

```diff
@@ -409,8 +409,12 @@ def by_group(cls, group, wrap=True):
 
     @classmethod
     def by_data_type(cls, domain, data_type, bypass_cache=False):
-        data_type_id = _id_from_doc(data_type)
-        return get_fixture_items_by_data_type(domain, data_type_id, bypass_cache)
+        return cls.by_data_types(domain, [data_type], bypass_cache)
+
+    @classmethod
+    def by_data_types(cls, domain, data_types, bypass_cache=False):
+        data_type_ids = frozenset(_id_from_doc(d) for d in data_types)
```

Comment:
We could add support for frozen set to that I suppose—I think it's just never come up before